act-rules-format.md

```diff
@@ -179,33 +179,76 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
 </div>
 
 <aside class=example>
-  <header>Example accessibility requirements mapping for a rule that tests if an image button has an accessible name:</header>
+  <header>Example accessibility requirements mapping for a rule that tests if a form field has a non-empty accessible name:</header>
   <blockquote><ul>
     <li>
-      [Success criterion 1.1.1: Non-text content](https://www.w3.org/TR/WCAG21/#non-text-content)
+      [Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)
       <ul>
         <li>**Required for conformance** to WCAG 2.0 and WCAG 2.1 level A and higher</li>
-        <li>Outcome mapping: <ul>
+        <li>Outcome mapping:<ul>
           <li>Any `failed` outcomes: not satisfied</li>
           <li>All `passed` outcomes: further testing is needed</li>
           <li>An `inapplicable` outcome: further testing is needed</li>
         </ul></li>
       </ul>
     </li>
+  </ul></blockquote>
+</aside>
+
+### Mapping Secondary Requirements ###
+
+An ACT Rule <em class="rfc2119">may</em> show that the outcomes of the rule have related or secondary [=accessibility requirements=] for that test subject. The secondary accessibility requirements may not be required for conformance to the [=accessibility requirements document=]. When one or more of the outcomes for a test target is `failed`, the secondary accessibility requirement could be <dfn>not satisfied</dfn> for the test subject or <dfn>further testing is needed</dfn> . When all of the outcomes are `passed` or `inapplicable`, the secondary accessibility requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>. The rule must explain the secondary relationship of the accessibility requirement to the rule. 
+
+<aside class=example>
+  <header>Example accessibility requirements mapping for a rule that tests if text has minimum contrast:</header>
+  <blockquote><ul>
     <li>
-      [Success criterion 4.1.2: Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)
+      [Success criterion 1.4.3 Contrast Minimum (AA)](https://www.w3.org/TR/WCAG21/#contrast-minimum)
       <ul>
         <li>**Required for conformance** to WCAG 2.0 and WCAG 2.1 level A and higher</li>
         <li>Outcome mapping:<ul>
           <li>Any `failed` outcomes: not satisfied</li>
-          <li>All `passed` outcomes: further testing is needed</li>
+          <li>All `passed` outcomes: satisfied</li>
+          <li>An `inapplicable` outcome: further testing is needed</li>
+        </ul></li>
+      </ul>
+    </li>
+    <li>
+      [Success Criterion 1.4.6 Contrast (Enhanced) (Level AAA)](https://www.w3.org/TR/WCAG21/#contrast-enhanced)
+      <ul>
+        <li>**Secondary** to WCAG 2.0 and WCAG 2.1 level A and higher</li>
+        <li>Outcome mapping:<ul>
+          <li>Any `failed` outcomes: not satisfied</li>
+          <li>All `passed` outcomes: further testing is needed; SC 1.4.6 requires higher contrast ratios than SC 1.4.3</li>
```

Comment:
It feels like we should formalize how reasoning behind a secondary requirement is given. In this scenario the reasoning is fairly straight forward, in others this might not be the case. Secondary requirements might need a sub-section or something to hold this reasoning.

Reply:
add a section in Background to explain secondary requirements